python/deadlinesubmissionsettings/app.py

```diff
@@ -29,14 +37,28 @@ def setup_ui(self):
 
         publish = QtWidgets.QCheckBox("Suspend Publish Job")
         defaultlayer = QtWidgets.QCheckBox("Include Default Render Layer")
+        run_slap_comp = QtWidgets.QCheckBox("Run Slap Comp")
+
+        brows_hlayout = QtWidgets.QHBoxLayout()
```

Comment:
Hehe, sorry @aardschok - noticed just right now there are actually much more variables with `brows` instead of `browse` - this line and the lines below (45 to 54, 120, 131)